### DISCUSSION TITLE
Map PRs that need an extra eye

### DISCUSSION THREAD

**Discussion by CharlieAtlas** [triplea-game/triplea#8499] [General]
This seems like a better place for these.  I will close the issue i made them in.  

From https://github.com/triplea-game/triplea/issues/8245

Territory effects images were changed, and I don't know how to test them.  
- [ ] triplea-maps/stellar_forces#3
- [ ] triplea-maps/another_world#2

I didn't have permission for these
- [x] triplea-maps/battle_for_arda#3
- [x] triplea-maps/steampunk_advanced#1

Created by me, so i feel like someone else should look at them before merging them.
- [x] triplea-maps/domination_1914_no_mans_land#10
- [ ] triplea-maps/world_war_ii_global#42
- [x] https://github.com/triplea-maps/global_war/pull/2
- [x] https://github.com/triplea-maps/red_sun_over_china/pull/16

Beyond my knowledge
- [ ] triplea-maps/1914-cow-empires#1

**Comment by DanVanAtta** [comment]
My 2 cents, this actually might make a better issue. It's more of a work item, closing it is appropriate when done, and we are not really discussing anything here other than tracking work to be done.

What's the background of these PRs, have they just been hanging out?

**Comment by CharlieAtlas** [comment]
Seeing if replying to email works along with everything else i say after
this.

It has been an issue for a little while, and nothing happened with it after
one reply, so i thought this might be better. Also, by and large any issue
with maps that i ask questions about or say anything in also tends to go by
the wayside. I think they just get inundated too quickly to get good
attention, and there isn't a maps team.

Some have been hanging out for years.
A few are ones that i have committed, so i feel that someone else should
merge those.

My main hope was that if i just got them some attention, it should take
just 10 minutes for someone else to decide they can be merged and merge
them.

**Comment by DanVanAtta** [comment]
We are stretched thin, getting tasks done and behind us is really important. 
This is good follow-up @CharlieAtlas

**Comment by DanVanAtta** [comment]
I merged a couple, checked them off the list, and would like more info about the domination 1914 change @CharlieAtlas , please have a look at the PR. Please 'bump' this topic  if I don't come back to this in the next few days.

**Comment by CharlieAtlas** [comment]
Moved to a TripleA-maps project.  How do we close a discussion?
